packages/outstatic/src/utils/server.ts

```diff
@@ -2,6 +2,17 @@ import fs from 'fs'
 import { join } from 'path'
 import matter from 'gray-matter'
 
+export interface IOptions {
+  status: 'all' | 'published' | 'draft'
+}
+
+const defaultOptions: IOptions = {
+  status: 'published'
+}
+
+const isCorrectStatus = (options: IOptions, status?: string) =>
```

Comment:
This signature does not reflect the method name. Document as first prop, status as second. Simplifies the check to an array includes

```ts
const containsStatus= (document: Record<string, unknown>, status: PostStatus[]) => {
  return status.includes(document.status)
}
```